frontend/src/js/components/devices/dialogs/DeviceConnectionDialog.tsx

```diff
@@ -86,66 +86,68 @@ const OnPremDeviceConnectionExplainer = ({ isEnterprise }) => (
-          <div className="flexbox center-aligned">
-            <Typography variant="subtitle1" gutterBottom>
-              Use a virtual device
-            </Typography>
-            <img src={docker} className={classes.virtualLogo} />
+          <div className="flexbox space-between">
+            <div className="flexbox centered">
+              <img src={zephyr} className={classes.zephyrLogo} />
+              <Typography variant="subtitle1" gutterBottom>
+                Zephyr MCU
+              </Typography>
+            </div>
+            <Chip size="small" label="Micro" />
           </div>
-          <Typography variant="body1">
-            Don&apos;t have a Raspberry Pi?
-            <br />
-            You can use our Docker-run virtual device to go through the same tutorial.
-          </Typography>
+          <Typography variant="body1">Connect an Espressif ESP32-S3 DevKitC, or any compatible microcontroller that supports MCUBoot in Zephyr.</Typography>
           <div>
-            <Typography variant="body1" color="text.secondary">
-              If you want to evaluate our commercial components such as mender-monitor, please use a physical device instead as the virtual client does not
-              support these components at this time.
-            </Typography>
-            <Button variant="text" size="small" onClick={() => setVirtualDevice(true)}>
-              Try a virtual device
+            <Button variant="text" size="small" endIcon={<ArrowForwardIcon />} onClick={() => setMcu(true)}>
+              Get started with Zephyr
             </Button>
           </div>
         </div>
         <div className={`padding-small ${classes.deviceSection}`}>
           <Typography variant="subtitle1" gutterBottom>
-            Other devices
+            Don&#39;t have a device?
           </Typography>
-          <Typography variant="body1">See the documentation to integrate the following with Mender:</Typography>
-          <List>
-            {docsLinks.map(item => (
-              <ListItem key={item.key} disablePadding className="padding-top-none padding-bottom-none">
-                <DocsLink path={item.target} title={item.title} />
-              </ListItem>
-            ))}
-          </List>
-          <MenderHubReference />
+          <Typography variant="body1">
+            You can use our virtual device to explore the UI, deploy an update and get a quick feel for the features of Mender.
+          </Typography>
+          <div className="flexbox margin-top-small">
+            <Button variant="text" size="small" onClick={() => setVirtualDevice(true)} endIcon={<ArrowForwardIcon />}>
+              Try the virtual device
+            </Button>
+          </div>
         </div>
       </div>
+      <Typography variant="body1" className={classes.bottomText}>
+        <a href="https://docs.mender.io/overview/device-support" target="_blank" rel="noopener noreferrer">
+          Visit our documentation{' '}
+        </a>
```

Comment:
Sorry that I overlooked this earlier, please use the `DocsLink` component as this can give us some more usage data

```suggestion
        <DocsLink path="overview/device-support" title="Visit our documentation" />{' '}
```